app/src/main/res/values/arrays.xml

```diff
@@ -46,6 +47,7 @@
         <item>@drawable/ic_theme_dark</item>
         <item>@drawable/ic_theme_ford</item>
         <item>@drawable/ic_theme_bmw</item>
+        <item>@drawable/ic_theme_mk</item>
```

Comment:
Put a reference to the preview image in the `ThemesThumbs` array. You specify it with `@drawable/ic_theme_<your theme name>` matching the file created for the thumbnail without the extension.